### DISCUSSION TITLE
why Copilot not activate ?

### DISCUSSION THREAD

**Discussion by MiithRanDir** [community/community#41954] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

![image](https://user-images.githubusercontent.com/5079543/208225693-26e4d8b0-c571-474d-9694-7554a799829d.png)
Version: 1.70.0-insider (user setup)
Commit: 3cbf3063281410d21951877eb628551809d70c6b
Date: 2022-07-25T05:27:52.229Z
Electron: 18.3.5
Chromium: 100.0.4896.160
Node.js: 16.13.2
V8: 10.0.139.17-electron.0
OS: Windows_NT x64 10.0.19045

my extension github copilot not activate

**Comment by horridharry** [comment]
[Click this link](https://github.com/community/community/discussions/41962#discussioncomment-4435896) to refer to my guidance on how to resolve this issue.

**Comment by ICT154** [comment, marked as answer]
try to do the following steps:

1. Open the Extensions panel again in the same way as above.
2. Click on the "More actions" icon (three horizontal dots) next to the "GitHub Copilot" extension name.
3. Select the "Install another version" option from the menu that appears.
4. Select a different version of the "GitHub Copilot" extension than the one you currently have installed, then click the "Install" button.
5. After the installation process is complete, restart your Visual Studio Code.

i also had the same problem like this this morning, and it worked fine
![Screenshot (132)](https://user-images.githubusercontent.com/44914917/208228657-42fb7202-99ef-4174-b4e9-4b7546c8e472.png)

**Reply by fzaninotto** [reply]
Works great, thanks! It seems Copilot version 1.64.7694, which is the latest available version at the time of writing, is buggy. Installing the previous version solves the problem.
